File: scalechain-net/src/test/scala/io/scalechain/blockchain/net/handler/AddrMessageHandlerSpec.scala
package io.scalechain.blockchain.net.handler

import java.io.File

import io.netty.channel.embedded
import io.netty.channel.embedded.EmbeddedChannel
import io.scalechain.blockchain.chain.{ChainTestDataTrait, BlockchainTestTrait}
import org.scalatest._

class AddrMessageHandlerSpec extends MessageHandlerTestTrait with ShouldMatchers {
  this: Suite =>

  val testPath = new File("./target/unittests-AddrMessageHandlerSpec/")

  "handle" should "" in {
  }
}

File: scalechain-net/src/test/kotlin/io/scalechain/blockchain/net/handler/AddrMessageHandlerSpec.kt
package io.scalechain.blockchain.net.handler

import io.kotlintest.KTestJUnitRunner
import io.kotlintest.matchers.Matchers
import org.junit.runner.RunWith
import java.io.File

@RunWith(KTestJUnitRunner::class)
class AddrMessageHandlerSpec : MessageHandlerTestTrait(), Matchers {

  override val testPath = File("./build/unittests-AddrMessageHandlerSpec/")

  init {
    "handle" should "" {
    }
  }
}